src/css/components/ProgressBar.scss

```diff
@@ -1,135 +1,301 @@
   }
-  .ProgressBar--xlarge & {
-    border-radius: $border-radius-large * 2;
+}
+
+.ProgressBar--large {
+  height: $su-medium;
+  &, .ProgressBar-status {
+    border-radius: $su-medium / 2;
   }
 }
-// inline title styles
-.ProgressBar--inline {
-  display: flex;
-  width: 100%;
-  align-items: center;
-  .ProgressBar-title {
-    width: auto;
-    margin-right: $su-small;
+
+.ProgressBar--xlarge {
+  height: $su-large;
+  &, .ProgressBar-status {
+    border-radius: $su-large / 2;
   }
 }
 
-// radial progress bar
-.ProgressBar--radial {
+// * 2. Vertical Progress Modifier
+
+.ProgressBar--vertical {
+  height: 100%;
+  width: $su-xlarge;
   position: relative;
-  display: block;
-  height: auto;
-  width: 100%;
-  margin-bottom: $su-medium;
-  .circle {
-    clip: rect(0, scaleGrid(7), scaleGrid(7), scaleGrid(7)/2 );
-    left: 0;
+
+  .ProgressBar-status {
+    height: auto; // overidden inline
+    width: 100%;
     position: absolute;
-    top: 0;
-    height: scaleGrid(7);
-    width: scaleGrid(7);
+    bottom: 0;
+    left: 0;
   }
-  .ProgressBar-range {
-    height: scaleGrid(7);
-    width: scaleGrid(7);
-    background: transparent;
-    border-radius: 50%;
-    &:before {
-      content: '';
-      display: block;
-      width:  100%;
-      height: 100%;
-      border: $su-small solid $cu-background;
-      box-shadow: inset 0 0 1.5px rgba($ts-black, .25);
-      border-radius: 50%;
-    }
-    .ProgressBar-status {
-      position: absolute;
-      top: 0;
-      left: 0;
-      clip: rect(0, scaleGrid(7)/2, scaleGrid(7), 0);
-      height: inherit;
-      width: inherit;
-      background: transparent;
-      border: $su-small solid $cu-positive;
-      border-radius: 50%;
-      transform: rotate(0);
+
+  &, .ProgressBar-status {
+    border-radius: $su-xxlarge / 2;
+  }
+
+}
+
+// * 2a. Vertical Progress Sizes
+
+.ProgressBar--vertical {
+
+  &.ProgressBar--xsmall {
+    width: $su-medium;
+    &, .ProgressBar-status {
+      border-radius: $su-medium / 2;
     }
   }
-  &.ProgressBar--large .ProgressBar-range {
-    width: scaleGrid(10);
-    height: scaleGrid(10);
-    &:before {
-      border-width: $su-medium;
+
+  &.ProgressBar--small {
+    width: $su-large;
+    &, .ProgressBar-status {
+      border-radius: $su-large / 2;
     }
-    .ProgressBar-status {
-      border-width: $su-medium;
-      clip: rect(0, scaleGrid(10)/2, scaleGrid(10), 0);
+  }
+
+  &.ProgressBar--large {
+    width: scaleGrid(6);
+    &, .ProgressBar-status {
+      border-radius: scaleGrid(6) / 2;
     }
-    .circle {
-      clip: rect(0, scaleGrid(10), scaleGrid(10), scaleGrid(10)/2 );
-      width: inherit;
-      height: inherit;
-      &.whole {
-        clip: rect(auto, auto, auto, auto)
-      }
+  }
+
+  &.ProgressBar--xlarge {
+    width: scaleGrid(8);
+    &, .ProgressBar-status {
+      border-radius: scaleGrid(8) / 2;
     }
+  }
+
+}
+
+// * 3. Precise Modifier
+
+.ProgressBar.ProgressBar--precise {
```

Comment:
Can this just be `.ProgressBar--precise`?

Reply:
needed it for specificity

Reply:
..actually nevermind, updated it. Was running into a specificty issue but not an issue. also moved this to a new PR : https://github.com/teamsnap/teamsnap-ui/pull/74